L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -38,4 +38,3 @@
                                                TrackQuality =cms.bool(False),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
-
```

Comment:
it seems like it is a nonsense change in the L1Trigger/TrackFindingTracklet package. Can it please be removed to avoid confusion about PRs that made changes to this package? thanks.

Reply:
Fixed